packages/workspace/src/generators/utils/get-npm-package-version.ts

```diff
@@ -0,0 +1,23 @@
+export function getNpmPackageVersion(packageName: string): string | null {
+  try {
+    let version;
```

Comment:
Can avoid the `let` with

```
const version = require('child_process').execSync(
  `npm view ${packageName} version`,
  { stdio: ['pipe', 'pipe', 'ignore'] }
);
```

And the `null` return could be combined:

```
try {
  // ...
 
  if (version) {
    return ...
  }
catch {
 // ignore
}
return null;
```

Less returns to track -- even though this is a small function.

Reply:
done!